packages/pds/tests/subscription/repo.test.ts

```diff
@@ -138,7 +143,10 @@ describe('sync', () => {
       forSnapshot(originalTl.data.feed),
     )
 
-    await expect(getTableDump()).resolves.toEqual(originalTableDump)
+    // Permissive of indexedAt times changing
```

Comment:
Reprocessing an update does update the `indexedAt` time.  Ideally I think it wouldn't, but I'm not sure if it's really worth adding the check to work around.